src/js/health-records/containers/UpdatePage.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import { checkRefreshStatus } from '../actions/refresh';
+import { submitForm } from '../actions/form';
+
+import ProgressBar from '../../common/components/ProgressBar';
+
+export class UpdatePage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.pollRefresh = () => {
+      setInterval(() => {
+        props.checkRefreshStatus();
+      }, 10000);
+    };
+    this.handleSkipToDownload = this.handleSkipToDownload.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.checkRefreshStatus();
+    this.pollRefresh();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const erroredUpdates = nextProps.refresh.statuses.ERROR;
+    if (erroredUpdates.length === 0) {
```

Comment:
Is it possible for the first refresh check to have no errors in the returned data? What if the first response is not up-to-date, but has status 'OK' on all of the data?